transmogrifier/sources/xml/datacite.py

```diff
@@ -54,7 +55,7 @@ def get_optional_fields(self, xml: Tag) -> dict | None:
                 if self.valid_content_types([content_type]):
                     fields["content_type"] = [content_type]
                 else:
-                    return None
+                    raise SkippedRecordEvent(source_record_id)
```

Comment:
Looks good.  But see comment above about adding messaging close to the error/situation, confident it would get logged higher up.